### DISCUSSION TITLE
Exporting bookmarks from desktop app to Android app

### DISCUSSION THREAD

**Discussion by cjcfojc** [AlexandreRouma/SDRPlusPlus#995] [Q&A]
Good morning...I'm definitely new here and still finding my way around, but I had a quick question.  I'm (very) successfully running the latest nightly on Ubuntu 22.04.  I've spent quite a bit of time creating and organizing various bookmark lists for all the signals and traffic in my area.  On the side, I've been working on a mini "cyberdeck" of sorts that lives inside of a Pelican 1040 case (battery, RTL LNA, RTL SDR, battery pack, etc) and it runs off of an unlocked and gutted Galaxy S20 Ultra.  Just like with the desktop version, SDRPP runs wonderfully on the S20, BUT I'm left with a question...I'd like to transfer/copy my freq manager setup from desktop to the mobile setup and I haven't quite figured that out yet.  I'd appreciate some guidance and thanks for all the work that's been put into this software.  Thanks!

**Comment by AlexandreRouma** [comment]
No way to do it currently without using ADB commands

**Reply by cjcfojc** [reply]
Well, I am definitely willing to try and figure it out.  I've got a bit of time behind ADB as well after clearing the junk from that S20.  you have a recommended starting point?
